sql/moz-fx-data-shared-prod/telemetry_derived/latest_versions/metadata.yaml

```diff
@@ -1,7 +1,6 @@
 friendly_name: Latest Versions
 workgroup_access:
 - role: roles/bigquery.dataViewer
-  members:
+  members: ~
```

Comment:
~?

Reply:
null in YAML, should be the same as empty